go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	github.com/streamrail/concurrent-map v0.0.0-20160823150647-8bf1e9bacbf6 // indirect
 	github.com/stretchr/testify v1.3.0
 	github.com/vishvananda/netlink v1.0.0
-	github.com/vmware/octant v0.8.0
+	github.com/vmware/octant v0.0.0-00010101000000-000000000000
```

Comment:
Cannot be changed due to issue: #111 , please take a look at this patch https://github.com/vmware-tanzu/antrea/pull/268